src/main/java/ch/bfh/bti7081/view/SeminarView.java

```diff
@@ -203,12 +210,17 @@ private void setActions() {
             newSeminar.getUI().ifPresent(ui -> ui.navigate("seminar/new"));
         });
 
+        // close details action
+        closeDetails.addClickListener(event -> {
+            details.close();
```

Comment:
Würde hier grundsätzlich noch ein seminarGrid.deselectAll(); hinzufügen, damit es keine Fehlermeldung mehr gibt wenn man nach dem Öffnen eines Detail-Fensters den Filter umstellt. 
Falls Schliessen auch ohne Button möglich ist: Es gibt für den Dialog OnClose Listener, dort könnte man das auch integrieren. 😄

Reply:
das habe ich bereits probiert. bei mir hats noch nicht sauber geklappt. deshalb habe ich dies hier vorerst mal so eingefügt. die thematik mit mit den "close-Listeners" werde ich in einem separaten branch (inkl. verbesserung seminarcategory und seminarfilter) lösen. dieser folgt bald.